src/main/java/uk/ac/ebi/embl/flatfile/writer/FlatFileWriter.java

```diff
@@ -227,4 +229,8 @@ public void setForceLineBreak(boolean forceLineBreak) {
 	public void setOptimalLineLength(int optimalLineLength) {
 		this.optimalLineLength = optimalLineLength;
 	}
+
+	public void setCustomMaximumLineLength(Integer customMaximumLineLength) {
```

Comment:
would just call this:
```
setMaximumLineLength
```
like the
```
setOptimalLineLength
```
One of these is not more custom than the other.